src/components/Staking/StakingHowSoloWorks.tsx

```diff
@@ -56,10 +45,13 @@ const StakingHowSoloWorks: React.FC<IProps> = () => {
   ]
 
   return (
-    <Flex>
+    <Center
+      flexDirection={{ sm: "column", md: "row" }}
```

Comment:
Recommend your first value use the `base` token instead of `sm` to ensure the value is applied at all screen widths below `md`

The layout is reverting back to row when the width gets below 480px.
```suggestion
      flexDirection={{ base: "column", md: "row" }}
```

Reply:
@TylerAPfledderer sounds good ill commit the suggestion nice catch that's on me.